src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/tabs/fullScript_pending.properties

```diff
@@ -1,21 +1,16 @@
-pendingScripts=Scripts pending approval
-
 # pending scripts
 ## table headers
-pendingHash=Hash
-pendingHash_tooltip=The hash could be seen as a signature of the script. It's used as an identifier to reference the content of the script.
 pendingExpandCode=Code
 pendingLanguage=Language
-pendingScriptLength=Length
-pendingScriptLength_tooltip=Length of the script in characters
 contextUser=Requester
 contextItem=Context
 requestApprovalTime=Request date
 requestApprovalTime_tooltip=Inform about when the approval request was made.
 
 ## table content
 noRequesterSinceMetadata=N/A
-noRequesterSinceMetadata_tooltip=The requester was not recorded. It could be due to the request being created before the metadata introduction or the code calling this script did not provide user information.
+noRequesterSinceMetadata_tooltip=The requester was not recorded. \
+  It could be due to the request being created before the metadata introduction or the code calling this script did not provide user information.
```

Comment:
```suggestion
  It could be because the request was created before the metadata introduced or the code calling this script did not provide user information.
```

Reply:
> before the metadata introduced

Ungrammatical?

Reply:
Note to self: copy+paste is my friend. Fixed.